src/debugger/debug_workflow.ts

```diff
@@ -113,10 +113,17 @@ export class WorkflowContextDebug extends DBOSContextImpl implements WorkflowCon
 
     const result = await this.#dbosExec.userDatabase.transaction(wrappedTransaction, txnInfo.config);
 
+    // If returned nothing and the recorded value is also null/undefined, we just return it
+    if (result === undefined && !check!.output) {
```

Comment:
what is the `!` *before* check mean?

Reply:
Never mind, it's not. the use of ! for both not and "ignore maybe null" threw me

Reply:
Maybe explicitly compare `check!.output` to false to make this clearer?